Test/dafny0/LabelsOldAt.dfy

```diff
@@ -83,4 +83,196 @@ class C {
     case true =>
       assert unchanged@End({c,d});
   }
+
```

Comment:
I find it difficult to say whether the test-cases cover the functionality. I'm wondering if it would be useful to structure the test-cases according to the documentation, or even to have a file where the two are intertwined.

Reply:
Running the tests and doing some coverage analysis of the sources would also help (and would surely find gaps).